Modules/Core/Common/src/itkObjectFactoryBase.cxx

```diff
@@ -832,8 +832,7 @@ ObjectFactoryBase::SynchronizeObjectFactoryBase(void * objectFactoryBasePrivate)
 std::list<ObjectFactoryBase *>
 ObjectFactoryBase::GetRegisteredFactories()
 {
-  //  static SingletonIndex * singletonIndex = SingletonIndex::GetInstance();
-  //  Unused(singletonIndex);
+  //  [[maybe_unused]] static SingletonIndex * singletonIndex = SingletonIndex::GetInstance();
```

Comment:
This commented-out declaration of `singletonIndex` was introduced with pull request #118 commit a66337ec215c88f4900a2caf419b055483c42085 "ENH: Synchronize factories across modules in Python", Francois Budin (@fbudin69500), February 2019:

https://github.com/InsightSoftwareConsortium/ITK/blob/a66337ec215c88f4900a2caf419b055483c42085/Modules/Core/Common/src/itkObjectFactoryBase.cxx#L925-L926

Maybe we might as well just remove those two commented-out lines of code. It's very hard to keep commented-out code up-to-date. What do you think?

Reply:
Since you already did the work, perhaps let's keep it this time. But yes, we can occasionally clean up commented-out code.